src/main/tut/index.html

```diff
@@ -99,12 +101,16 @@
 
 --
 
-Better, but not great: you have to explicitly specify how to decode types every time.
+<!-- COMMENT -->
+
+Better, but not great: one has have to manually convert the value to the desired type every time.
 
 ---
 
 ## Small detour: implicits
 
+<!-- COMMENT: using the word implicit to describe implicits seems sub-optiomal? -->
+
 > When a function expects a parameter of type `T` _and_ that parameter is marked as implicit _and_ there exists a
 > value of type `T` marked as implicit in scope, then the compiler can use that value automatically.
```

Comment:
These two lines are interesting. What you're saying is accurate, but as a reader, it seems odd to describe a term by using it? `implicit` is used twice in this sentence, but the term is never explained. Probally a couple of ways to go about this:

1. Either provide verbal guidance and use the formal definition as a prop on the slides,
2. Word the slide so it can be read by non-attendees after the fact, and verbally expand on it as needed. 

I'm inclined to recommend `2`, simply because having it as-is I would worry about the presenter simply reading the statement aloud to the audience.